lib/API/JsonApi/v1/Links.php

```diff
@@ -108,7 +108,7 @@ class Links implements LoggerAwareInterface, JsonApiResponseInterface
      */
     public function __construct()
     {
-
+        $this->logger = new Logger();
```

Comment:
Not sure about hardcoding the concrete class here. Kind of defies the purpose of using an interface in the first place. 

Should we not rather call setLogger() from the appropriate place?